src/views/Dashboard.jsx

```diff
@@ -0,0 +1,311 @@
+
+          <div className="flex flex-1 gap-x-4 self-stretch lg:gap-x-6">
+            <div className="relative flex flex-1"></div>
+            <div className="flex items-center gap-x-4 lg:gap-x-6">
+              {/* Separator */}
+              <div
+                className="hidden lg:block lg:h-6 lg:w-px lg:bg-gray-200"
+                aria-hidden="true"
+              />
+
+              {/* Profile dropdown */}
+              <Menu as="div" className="relative">
+                <Menu.Button className="-m-1.5 flex items-center p-1.5">
+                  <img
+                    className="h-10 w-10 rounded-full bg-gray-50 object-cover"
+                    src={
+                      currentUser.photoURL
+                        ? currentUser.photoURL
+                        : "https://res.cloudinary.com/yilin1234/image/upload/v1692746214/istockphoto-1337144146-612x612_rtbsc1.jpg"
+                    }
+                    alt="User Profile"
+                  />
+                  <span className="hidden lg:flex lg:items-center">
+                    <span
+                      className="ml-4 text-sm font-semibold leading-6 text-gray-900"
+                      aria-hidden="true"
+                    >
+                      {currentUser.displayName ||
+                        currentUser.email.split("@")[0]}
+                    </span>
+                    <ChevronDownIcon
+                      className="ml-2 h-5 w-5 text-gray-400"
+                      aria-hidden="true"
+                    />
+                  </span>
+                </Menu.Button>
+                <Transition
+                  as={Fragment}
+                  enter="transition ease-out duration-100"
+                  enterFrom="transform opacity-0 scale-95"
+                  enterTo="transform opacity-100 scale-100"
+                  leave="transition ease-in duration-75"
+                  leaveFrom="transform opacity-100 scale-100"
+                  leaveTo="transform opacity-0 scale-95"
+                >
+                  <Menu.Items className="absolute right-0 z-10 mt-2.5 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-gray-900/5 focus:outline-none">
+                    {/* Your Profile Item */}
+                    <Menu.Item>
+                      {({ active }) => (
+                        <Link
+                          to="my-profile" // Replace with your desired link
+                          className={classNames(
+                            active ? "bg-gray-50" : "",
+                            "block px-3 py-1 text-sm leading-6 text-gray-900"
+                          )}
+                        >
+                          your profile
+                        </Link>
+                      )}
+                    </Menu.Item>
+                    {/* Sign Out Item */}
+                    <Menu.Item>
+                      {({ active }) => (
+                        <div
```

Comment:
I see it's been mentioned already but I agree I think it would be best to have the sign out as a button rather than a div.